lib/cli/github_settings/do_set_branch_protection.js

```diff
@@ -333,12 +401,13 @@ do_set_branch_protection.help = [
     'to pass before allowing PRs to be merged. For example:',
     '    jr gh set-branch-protection -s continuous-integration/jenkins/pr-head,integration-approval REPO',
     '',
-    'The `-U` option can be used to "unlock" the master branch for direct git',
+    'The `-U` option can be used to "unlock" the branches for direct git',
```

Comment:
I also would have been fine with `-U` *only* unlocking the "master" branch as that is the only one we anticipate having a need for unlocking. Whichever is cleaner/easier.

Reply:
Maybe. I could imagine cases where we _might_ want to allow a non-squash merge of a bunch of of changes that were ported from a mantav2 repository in order to retain those separate commits, though yes, perhaps that's a bit bit far-fetched. I'll _also_ admit it's easier to have -U operate on both branches, so will keep it as is.

Taking this easier route is slightly more disruptive on the repository in that we're locking two branches rather than one, but it's not like these are particularly high-traffic repositories, so it's probably not a big deal.